controllers/neutronapi_controller.go

```diff
@@ -528,7 +528,7 @@ func (r *NeutronAPIReconciler) reconcileNormal(ctx context.Context, instance *ne
 	rbacRules := []rbacv1.PolicyRule{
 		{
 			APIGroups:     []string{"security.openshift.io"},
-			ResourceNames: []string{"anyuid", "privileged"},
+			ResourceNames: []string{"anyuid"},
```

Comment:
only once concern how we deal with extra mounts that allows host path https://github.com/openstack-k8s-operators/neutron-operator/blob/efd4186556fec42b73d22cfd13ae4dac8bb90488/api/bases/neutron.openstack.org_neutronapis.yaml#L1179-L1188 ?

Reply:
One option is to use this security context constraint instead:

```
hostmount-anyuid | Provides all the features of the restricted SCC, but allows host mounts and running as any UID and any GID on the system.
```

https://docs.openshift.com/container-platform/4.12/authentication/managing-security-context-constraints.html#default-sccs_configuring-internal-oauth

Reply:
@karelyatin (reviving a very old PR) I think that maybe it's a suggestion that we should rethink how we extend neutronapi service, instead of leaving hostmount enabled? Would e.g. switching to a PV with custom driver code be a better path here? Let's discuss it some time.